src/test/java/org/kohsuke/github/GHOrganizationTest.java

```diff
@@ -231,4 +231,30 @@ public void testCreateAllArgsTeam() throws IOException {
         assertThat(team.getDescription(), equalTo("Team description"));
         assertThat(team.getPrivacy(), equalTo(GHTeam.Privacy.CLOSED));
     }
+
+    @Test
+    public void testAreProjectsEnabledForOrganization() throws IOException {
+        // Arrange
+        GHOrganization org = gitHub.getOrganization(GITHUB_API_TEST_ORG);
+
+        // Act
+        boolean result = org.areProjectsEnabledForOrganization();
+
+        // Assert
+        assertThat(result, is(true));
+    }
+
+    @Test
+    public void testSetProjectsEnabledForOrganization() throws IOException {
```

Comment:
Please use [nonRecordingGitHub()](https://github.com/hub4j/github-api/blob/b23934a5a15ccc4964a20a7ab422ee894dd9bbfb/src/test/java/org/kohsuke/github/AbstractGitHubWireMockTest.java#L230) to ensure the setting is `true` before starting, then toggle it to `false`.